spec/requests/task_search_by_tags_spec.rb

```diff
@@ -22,7 +25,9 @@
     let(:unexpected_tasks) { [uncompleted_task] }
 
     let(:expected_tasks) do
-      [create_task(author_id: task_author.id, title: 'a', tags: %w(tag), completed_at: :now, assignees: [assignee])]
+      [
+        create_task(author_id: task_author.id, title: 'a', tags: %w(tag), completed_at: :now, assignees: [assignee])
```

Comment:
__[rubocop]__
* Style/TrailingComma: Put a comma after the last item of a multiline array. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas)